src/service/repo/signedMessageRepo.ts

```diff
@@ -82,7 +85,7 @@ async function sendSignedMessage(
 
     return Promise.reject(response.data?.data);
   } catch (e) {
-    console.log(e);
+    console.log(e?.response);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging errors directly to the console can be unhelpful in a production environment. Consider using a logging service that provides more context and tracking capabilities. If this is not possible, at least provide some context around the error being logged.
```diff
-    console.log(e?.response);
+    console.error('Error sending signed message:', e?.response);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->